x-pack/plugins/cases/public/components/case_view/components/edit_tags.test.tsx

```diff
@@ -46,129 +38,121 @@ describe.skip('EditTags ', () => {
-    appMockRender.render(<EditTags {...defaultProps} />);
+    it('cancels on cancel', async () => {
+      appMockRender.render(<EditTags {...defaultProps} />);
 
-    userEvent.click(screen.getByTestId('tag-list-edit-button'));
+      userEvent.click(await screen.findByTestId('tag-list-edit-button'));
 
-    userEvent.type(screen.getByRole('combobox'), 'new{enter}');
+      userEvent.type(screen.getByRole('combobox'), 'new{enter}');
 
-    await waitFor(() => {
-      expect(screen.getByTestId('comboBoxInput')).toHaveTextContent('new');
-    });
+      expect(await screen.findByTestId('comboBoxInput')).toHaveTextContent('new');
+
+      userEvent.click(await screen.findByTestId('edit-tags-cancel'));
 
-    userEvent.click(screen.getByTestId('edit-tags-cancel'));
+      await waitFor(() => {
+        expect(onSubmit).not.toBeCalled();
+      });
 
-    await waitFor(() => {
-      expect(onSubmit).not.toBeCalled();
-      expect(screen.getByTestId('no-tags')).toBeInTheDocument();
+      expect(await screen.findByTestId('no-tags')).toBeInTheDocument();
     });
-  });
 
-  it('shows error when tag is empty', async () => {
-    appMockRender.render(<EditTags {...defaultProps} />);
+    it('shows error when tag is empty', async () => {
+      appMockRender.render(<EditTags {...defaultProps} />);
 
-    userEvent.click(screen.getByTestId('tag-list-edit-button'));
+      userEvent.click(await screen.findByTestId('tag-list-edit-button'));
 
-    await waitFor(() => {
-      expect(screen.getByTestId('edit-tags')).toBeInTheDocument();
-    });
+      expect(await screen.findByTestId('edit-tags')).toBeInTheDocument();
 
-    userEvent.type(screen.getByRole('combobox'), ' {enter}');
+      userEvent.type(screen.getByRole('combobox'), ' {enter}');
 
-    await waitFor(() => {
       expect(screen.getByText('A tag must contain at least one non-space character.'));
     });
-  });
 
-  it('shows error when tag is too long', async () => {
-    const longTag = 'z'.repeat(MAX_LENGTH_PER_TAG + 1);
+    it('shows error when tag is too long', async () => {
+      const longTag = 'z'.repeat(MAX_LENGTH_PER_TAG + 1);
 
-    appMockRender.render(<EditTags {...defaultProps} />);
+      appMockRender.render(<EditTags {...defaultProps} />);
 
-    userEvent.click(screen.getByTestId('tag-list-edit-button'));
+      userEvent.click(await screen.findByTestId('tag-list-edit-button'));
 
-    await waitFor(() => {
-      expect(screen.getByTestId('edit-tags')).toBeInTheDocument();
-    });
+      expect(await screen.findByTestId('edit-tags')).toBeInTheDocument();
 
-    userEvent.paste(screen.getByRole('combobox'), `${longTag}`);
-    userEvent.keyboard('{enter}');
+      userEvent.paste(screen.getByRole('combobox'), `${longTag}`);
+      userEvent.keyboard('{enter}');
 
-    await waitFor(() => {
-      expect(
-        screen.getByText('The length of the tag is too long. The maximum length is 256 characters.')
-      );
+      await waitFor(() => {
```

Comment:
Is the `await waitFor(...` still necessary?

Reply:
Thanks for catching this. I forgot them. I fixed it in [`26389a4` (#175763)](https://github.com/elastic/kibana/pull/175763/commits/26389a44238d0da6b0dddb9c7e0addc91e93525c)